### ISSUE TITLE
@NotNull method mobi/hsz/idea/gitignore/util/CacheMap.getParentStatus must not return null

### ISSUE THREAD

**Issue by jwhonce** [hsz/idea-gitignore#138]
Idea 14.1.3, build # 141.1010, plugin 1.1.2

Stack trace when I opened a go project:

```
    @NotNull method mobi/hsz/idea/gitignore/util/CacheMap.getParentStatus must not return null
    java.lang.IllegalStateException: @NotNull method mobi/hsz/idea/gitignore/util/CacheMap.getParentStatus must not return null
    	at mobi.hsz.idea.gitignore.util.CacheMap.getParentStatus(CacheMap.java:222)
    	at mobi.hsz.idea.gitignore.util.CacheMap.isFileIgnored(CacheMap.java:162)
    	at mobi.hsz.idea.gitignore.IgnoreManager.isFileIgnored(IgnoreManager.java:261)
    	at mobi.hsz.idea.gitignore.vcs.IgnoreFileStatusProvider.getFileStatus(IgnoreFileStatusProvider.java:68)
    	at com.intellij.openapi.vcs.impl.FileStatusManagerImpl.calcStatus(FileStatusManagerImpl.java:143)
    	at com.intellij.openapi.vcs.impl.FileStatusManagerImpl.getStatus(FileStatusManagerImpl.java:278)
    	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.getFileStatus(AbstractPsiBasedNode.java:115)
    	at com.intellij.ide.util.treeView.AbstractTreeNode.setForcedForeground(AbstractTreeNode.java:85)
    	at com.intellij.ide.util.treeView.AbstractTreeNode.postprocess(AbstractTreeNode.java:77)
    	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:91)
    	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:41)
    	at com.intellij.ide.util.treeView.AbstractTreeBuilder.updateNodeDescriptor(AbstractTreeBuilder.java:588)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$22.perform(AbstractTreeUi.java:944)
    	at com.intellij.ide.util.treeView.TreeRunnable.run(TreeRunnable.java:33)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.execute(AbstractTreeUi.java:1830)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.update(AbstractTreeUi.java:940)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.update(AbstractTreeUi.java:892)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.access$1600(AbstractTreeUi.java:62)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$24$1.run(AbstractTreeUi.java:1103)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$35.perform(AbstractTreeUi.java:1818)
    	at com.intellij.ide.util.treeView.TreeRunnable.run(TreeRunnable.java:33)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.execute(AbstractTreeUi.java:1830)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.execute(AbstractTreeUi.java:1815)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.access$5300(AbstractTreeUi.java:62)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$34.perform(AbstractTreeUi.java:1783)
    	at com.intellij.ide.util.treeView.TreeRunnable.run(TreeRunnable.java:33)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.executeYieldingRequest(AbstractTreeUi.java:2091)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.access$5900(AbstractTreeUi.java:62)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$38$1.perform(AbstractTreeUi.java:1983)
    	at com.intellij.ide.util.treeView.TreeRunnable.run(TreeRunnable.java:33)
    	at com.intellij.ide.util.treeView.AbstractTreeBuilder.runOnYeildingDone(AbstractTreeBuilder.java:433)
    	at com.intellij.ide.util.treeView.AbstractTreeUi.runOnYieldingDone(AbstractTreeUi.java:2186)
    	at com.intellij.ide.util.treeView.AbstractTreeUi$38.perform(AbstractTreeUi.java:1978)
    	at com.intellij.ide.util.treeView.TreeRunnable.run(TreeRunnable.java:33)
    	at com.intellij.ide.util.treeView.AbstractTreeBuilder$1.process(AbstractTreeBuilder.java:53)
    	at com.intellij.ide.util.treeView.AbstractTreeBuilder$1.process(AbstractTreeBuilder.java:50)
    	at com.intellij.util.containers.TransferToEDTQueue.processNext(TransferToEDTQueue.java:98)
    	at com.intellij.util.containers.TransferToEDTQueue.access$300(TransferToEDTQueue.java:36)
    	at com.intellij.util.containers.TransferToEDTQueue$1.run(TransferToEDTQueue.java:57)
    	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    	at java.awt.EventQueue.access$500(EventQueue.java:97)
    	at java.awt.EventQueue$3.run(EventQueue.java:709)
    	at java.awt.EventQueue$3.run(EventQueue.java:703)
    	at java.security.AccessController.doPrivileged(Native Method)
    	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:734)
    	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:569)
    	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
    	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by hsz** [comment]
@jwhonce Thank you for reporting. Fixed.

**Comment by hsz** [comment]
@jwhonce Fix has been already released with version v1.1.3 and is available for download.
